src/components/Book.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+
+function Book(props) {
+  return (
+    <Wrapper>
+      <Img>
```

Comment:
I would not name this `Img` as it lets people assume that this is an image when in fact it is not.

Reply:
Ok! Changed the name to `ImgContainer`, but I'm not sure if this is a better alternative. What do you think?

Reply:
That is way better 👍